### PR TITLE
feat(component): update apollo-server package version

### LINKED ISSUE #34: Upgrade package Apollo Server
Apollo Server 3 has significant changes in the implementation of graphql api structure.

### PR DESCRIPTION
**Pull request by rodrigodosanjosoliveira**
Fixes #34 

### Author/Contributor
- [ ] If documentation is needed for this change, has that been included in this pull request
- [ ] Remember to check if code coverage decrease, if so, please implement the necessary tests

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

File: test/defaultResolver.test.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ describe('GraphQL - Default Resolver', () => {
             // Then
             (err) => {
                 assert.strictEqual(err.name, 'UserInputError')
-                assert.strictEqual(err.invalidArgs, 'error')
+                assert.strictEqual(err.extensions.invalidArgs, 'error')
```
[the remark this answers is not in the document]
Comment (rodrigodosanjosoliveira) [reply test/defaultResolver.test.js:45]: Done

[the remark this answers is not in the document]
Comment (jhomarolo) [reply test/defaultResolver.test.js:45]: @rodrigodosanjosoliveira this should be a peer dependence and also check if is need to be inside dev as well

[the remark this answers is not in the document]
Comment (jhomarolo) [reply test/defaultResolver.test.js:45]: @rodrigodosanjosoliveira do you need some help with this pr?

[the remark this answers is not in the document]
Comment (rodrigodosanjosoliveira) [reply test/defaultResolver.test.js:45]: I put apollo-server packages as peer and dev dependencies

**Comment by jhomarolo** [comment]
@rodrigodosanjosoliveira please update the todo-list with this new version.

Thank you again for your contribution

**Pull request merged**